smurfs/src/components/Smurf.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+ const Smurf = (props) => {
+    return(
+        <div className="smurfs">
+        { props.smurf.map ( smurf => (
+                <div id={smurf.id} key={smurf.id}>
+                    <div>
+                    <p> {smurf.name} </p>
+                    <p> {smurf.age} </p>
+                    <p> {smurf.height} </p>
+                    </div>
+                    <button onClick={(event, id = smurf.id) => props.delete(event, id)} >Delete</button>
```

Comment:
you could have just passed in the event in the parameters here and added the anonymous function calls parameters: onClick={event => props.delete(event, smurf.id)}